mongo/integration/collection_test.go

```diff
@@ -1661,9 +1661,10 @@ func TestCollection(t *testing.T) {
 				mt.Fatalf("expected BulkWrite error %v, got %v", mongo.ErrUnacknowledgedWrite, err)
 			}
 		})
-		mt.Run("insert and delete with batches", func(mt *mtest.T) {
+		mt.RunOpts("insert and delete with batches", mtest.NewOptions().ClientType(mtest.Mock), func(mt *mtest.T) {
 			// grouped together because delete requires the documents to be inserted
-			numDocs := 100050
+			maxBatchCount := int(mtest.MockDescription.MaxBatchCount)
```

Comment:
No need to change, but IIUC, `MaxBatchCount` could be set to another value at this point. Monitoring and connection handshake is bypassed for the mock deployment, so this won't be checked until the operation runs.

Reply:
Yeah, it can be. Reducing the batch size would've been my next step if the test was still taking too long, but we got significant improvement already and I didn't feel the need to optimize further.